### DISCUSSION TITLE
Can I load script inside targeted element?

### DISCUSSION THREAD

**Discussion by itsmnthn** [nuxt/scripts#342] [Q&A]
There are certain embed that replaces the script tags on load to embed things. using useScript it appears in head tag right now. Is there a way to tell useScript to add script tag inside target element.

**Comment by harlan-zw** [comment]
`useScript` always injects to `<head>` (it's a head primitive). For embeds that need to live inside a specific element (e.g. Calendly's inline widget), some registries expose a `parentElement` option for this exact case. For arbitrary scripts that must mount inside a given DOM node, use `useHead({ script: [{ ..., tagPosition: 'bodyClose' }] })` or render a static `<script>` in your template. Closing 🙏
